docs/source/DEVELOPER.rst

```diff
@@ -144,3 +144,30 @@ The Deepcell Kiosk uses Google Kubernetes Engine to requisition resources on Goo
+
+2. There are three variables in the benchmarking pod's YAML file, ``conf/helmfile.d/0410.benchmarking.yaml``, that may need to be customized before benchmarking:
+
+    - ``MODEL`` is the model name and version that will be used in benchmarking. The model you choose should be present in the ``models/`` folder of your benchmarking bucket. See the `Van Valen Lab's benchmarking bucket <https://console.cloud.google.com/storage/browser/kiosk-benchmarking>`_ for an example.
+    - ``FILE`` is the name of the file that will be used for benchmarking. A file by this name should be in your benchmarking bucket in the ``uploads/`` folder.    
+    - ``COUNT`` specifies how many times the ``FILE`` will be submitted to the cluster for processing.
+
+3. Deploy a DeepCell Kiosk as you normally would. While navigating the cluster configuration menu, pay special attention to two configuration settings:
+
+    - The bucket name you provide should be that of the benchmarking bucket from step 1. To ensure the benchmarking process works properly, this bucket should not be attached to any other cluster you might have running.
+    - The Maximum Number of GPUs has a strong effect on benchmarking time, by effectively limiting how large the cluster can scale.
+
+4. Once the cluster has finished deploying, drop to the ``Shell`` via the DeepCell Kiosk main menu and create the benchmarking pod to begin the benchmarking process by executing the following command: ``kubectl scale deployment benchmarking --replicas=1``.
+
+5. Benchmarking jobs can take a day or more, depending on the conditions (# of images and max # of GPUs) chosen. To monitor the status of your benchmarking job, drop to the ``Shell`` within the DeepCell Kiosk main menu and execute the command ``stern benchmarking -s 10m``. This will show you the most recent log output from the `benchmarking` pod. When benchmarking has finished, the final line in the log should be ``Uploaded [FILEPATH] to [BUCKET] in [SECONDS] seconds.``, where ``[FILEPATH]`` is the location in ``[BUCKET]`` where the benchmarking data has been saved.
+
+6. Now that data has been generated for your benchmarking run and saved in your bucket, you can download and analyze it. Two top-level fields in this large JSON file that are probably of interest are:
+
+    - The exact running time of the benchmarking procedure is given in seconds as the value of the ``time_elapsed`` field.
+    - A slight underestimate of the total costs of the benchmarking run can be found as the value to the `total_node_and_networking_costs` field. (Note that the total_node_and_networking_costs does not include Storage Fees, Operation Fees, or Storage Egress Fees. These `were calculated <https://github.com/vanvalenlab/publication-figures/blob/383a90149eb86d4a0a697395edffb32d383bb1ca/figure_generation/data_extractor.py#L318>`_ after the fact by using the `Google Cloud guidelines <https://cloud.google.com/vpc/network-pricing#general>`_.)
```

Comment:
- The use of the pronoun "These" here is ambiguous (e.g. are we talking about the costs with or without the storage, operation, and egress fees?). Let's be explicit. 
- No need to use "Fees" 3 times in a list ("...Storage, Storage Egress, or Operation Fees.")
- "were calculated..." is another possible source of confusion. Should this be, "can be calculated..."?